web-ng/src/app/services/project/project.service.ts

```diff
@@ -36,12 +36,12 @@ export class ProjectService {
         this.activeProjectId$.pipe(
           // Asynchronously load project. switchMap() internally disposes
           // of previous subscription if present.
-          switchMap(id => this.dataStore.loadProject$(id))
+          switchMap(id => this.dataStore.loadProject$(id)),
+          shareReplay(1)
```

Comment:
Have you verified that `loadProject` is only getting called once? (eg add `doOnNext(p -> console.log(p))` instead loadProject$, for ex)? My guess is that it would called multiple times with `shareReplay` being placed here, but I could be wrong.